system-addon/lib/TopSitesFeed.jsm

```diff
@@ -374,6 +374,10 @@ this.TopSitesFeed = class TopSitesFeed {
   onAction(action) {
     switch (action.type) {
       case at.INIT:
+        // If the feed was previously disabled PREFS_INITIAL_VALUES was never received
+        if (!DEFAULT_TOP_SITES.length) {
+          this.refreshDefaults(this.store.getState().Prefs.values[DEFAULT_SITES_PREF]);
```

Comment:
Do defaults end up getting set twice in the common case of receiving INIT and PREFS_INITIAL_VALUES?

Reply:
Shouldn't. At `INIT` (before `PREFS_INITIAL_VALUES`) the `PrefsFeed` is not initialized and the store value will be `undefined` so `refreshDefaults` doesn't do anything.